app/views/shared/_cart_items.html.erb

```diff
@@ -19,9 +19,9 @@
         <% @items.each do |item| %>
         <tr class="event-row">
           <td>
-            <div class="rotate event-category">
+            <!-- <div class="rotate event-category">
               <p><%= item.event.category_name %></p>
-            </div>
+            </div> -->
```

Comment:
This was not lining up correctly. I didn't know how to fix it. We can either remove it, or we'll have to figure out how to get back where it's supposed to be